dev/docker/ocis.web.config.json

```diff
@@ -28,18 +28,7 @@
       "id": "preview",
       "path": "web-app-preview",
       "config": {
-        "mimeTypes": [
-          "image/tiff",
-          "image/bmp",
-          "image/x-ms-bmp"
-        ]
-      }
-    },
-    {
-      "id": "importer",
-      "path": "web-app-importer",
-      "config": {
-        "companionUrl": "https://host.docker.internal:9200/companion"
+        "mimeTypes": ["image/tiff", "image/bmp", "image/x-ms-bmp"]
```

Comment:
Not your code, but we have these three mime types hardcoded for thumbnails and for the preview app by now (see e.g. the `mimeTypes.ts` file in the preview app). So we can remove the `preview` app from the external apps section again and just add `preview` to the string-array under the `apps` key. Could you do that here? Only dev stack, so it doesn't need changelog anyway.